projects/Group_detr/group_detr/assigner.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import torch
+
+from mmdet.registry import TASK_UTILS
+from mmdet.models.task_modules.assigners.assign_result import AssignResult
+from mmdet.models.task_modules.assigners import BaseAssigner
+
+try:
+    from scipy.optimize import linear_sum_assignment
```

Comment:
We do not need to check it anymore because it is already in the requirements.txt